openshift-kube-apiserver/admission/customresourcevalidation/node/validate_node_config.go

```diff
@@ -25,18 +32,28 @@ var rejectionScenarios = []struct {
 	{fromProfile: configv1.LowUpdateSlowReaction, toProfile: configv1.DefaultUpdateDefaultReaction},
 }
 
-const PluginName = "config.openshift.io/RestrictExtremeWorkerLatencyProfile"
+const PluginName = "config.openshift.io/ValidateConfigNodeV1"
```

Comment:
not sure if there are any side effects of removing an existing plugin

Reply:
good point, I'm not sure either haha

Reply:
It looks like it should be okay ( :crossed_fingers: ), I think all of the custom resource validation plugins are exclusively configured as a unit in https://github.com/openshift/kubernetes/blob/a3f31a4f2c587420f1ddb3bcdb161bf64a8c436b/openshift-kube-apiserver/admission/customresourcevalidation/customresourcevalidationregistration/cr_validation_registration.go.